<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelSelectPositionMaps</class>
 <widget class="QWidget" name="PanelSelectPositionMaps">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
     <item>
      <widget class="WidgetTreeLocalMaps" name="treeViewLocalMaps">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="WidgetMapEditor" name="widgetMap"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxCoordinates">
       <property name="title">
        <string>Coordinates</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
            <item row="0" column="0">
             <widget class="QLabel" name="labelX">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxX">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelY">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBoxY">
              <property name="minimum">
               <number>-1000</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelYPlus">
              <property name="text">
               <string>Y plus:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBoxYplus">
              <property name="maximum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labelZ">
              <property name="text">
               <string>Z:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="spinBoxZ">
              <property name="maximum">
               <number>999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>268</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetMapEditor</class>
   <extends>QOpenGLWidget</extends>
   <header>widgetmapeditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTreeLocalMaps</class>
   <extends>QTreeView</extends>
   <header>widgettreelocalmaps.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
